app/controllers/rejected_orders_controller.rb

```diff
@@ -0,0 +1,25 @@
+class RejectedOrdersController < ApplicationController
+  def new
+    @order_client = OrderClient.find(params[:order_client_id])
+    @rejected_order = RejectedOrder.new
+  end
+
+  def create
+    @order_client = OrderClient.find(params[:order_client_id])
+    @rejected_order = @order_client.create_rejected_order(rejected_order_params)
+    @order_client.rejected!
+
+    if @rejected_order.save
+      flash[:notice] = 'Pedido rejeitado com sucesso'
+      redirect_to @order_client
```

Comment:
```suggestion
      redirect_to @order_client, notice: 'Pedido rejeitado com sucesso'
```

Aqui daria pra fazer algo assim pra simplificar a linha. Seria legal mover esse texto para o I18n o que acha?

Reply:
Concordo plenamente. Feito!